### ISSUE TITLE
wonderful project 

### ISSUE THREAD

**Issue by omgbox** [shimberger/gohls#47]
few issues 

- gohls does not detect new media in folder without restarting server for me. 

- ffmpeg supports , presets HLS encoding , 
   ultrafast , superfast ,veryfast , faster, fast , medium – default preset, slow , slower, very slow
   would be nice to have in the config these options.  - for the speed encoding issue, it might be 
   better to try ultrafast encoding to HLS , this is a similar project in node 
   https://github.com/mifi/hls-vod , which i used in past, it does not have any buffering encoding 
   issue,  might help you somehow. 

- encode multiple outputs in parallel 480,p 720p, 1080p, option to pick stream type. 

- detect srt subtitles in folder and convert to webvtt  
- detect subtitles inside MKV , encode together to HLS. 
- template options , view list, grid list ,
- small admin panel to control deletion , server settings , encoding etc.

**Comment by shimberger** [comment]
@omgbox  Thanks for all the ideas. I will not add an admin panel because I want to keep things simple. I created individual issues for the other bullets. I am closing this one.

**Comment by shimberger** [comment]
@omgbox New files should be added after a while through re-indexing. If this still does not happen, please open a bug ticket.

**Comment by shimberger** [comment]
@genofire I accidentially mentioned you in the new tickets instead of @omgbox. That was a brainfart on my part. Sorry for the spamming! It was not on purpose.

**Comment by genofire** [comment]
nice to know, that this project is still alive ;)

ATM i juse Jellyfin again and have not much time, so i do not know when i could take a look
